sdk/src/main/java/com/qonversion/android/sdk/internal/mappers/Mapper.kt

```diff
@@ -2,8 +2,11 @@ package com.qonversion.android.sdk.internal.mappers
 
 interface Mapper<T> {
     @Throws(IllegalStateException::class)
-    fun toMap(value: T): Map<String, Any?>
+    fun toMap(value: T): Map<String, Any?> { return emptyMap() }
```

Comment:
I don't like that it has default implementation, cause we might forget to add custom one on some mappers but call it then, getting empty map always. And also don't this we should implement it in every mapper. Maybe we'd better divide this in two different interfaces?

Reply:
For now, it's just a mock